src/main/java/org/jabref/gui/EntryTypeView.java

```diff
@@ -168,4 +178,82 @@ private void setEntryTypeForReturnAndClose(Optional<BibEntryType> entryType) {
         viewModel.stopFetching();
         this.close();
     }
+
+    public String getDescription(BibEntryType selectedType) {
+        EntryType entryType = selectedType.getType();
+        try {
+            StandardEntryType entry = (StandardEntryType) entryType;
+            switch (entry) {
```

Comment:
You can refactor this to use the new switch expresssions. 
https://blog.codefx.org/java/switch-expressions/